packages/helpers/src/dkim/index.ts

```diff
@@ -87,39 +103,15 @@ async function tryVerifyDKIM(email: Buffer | string, domain: string = "") {
   return dkimResult;
 }
 
-function getHeaderValue(email: string, header: string) {
-  const headerStartIndex = email.indexOf(`${header}: `) + header.length + 2;
-  const headerEndIndex = email.indexOf('\n', headerStartIndex);
-  const headerValue = email.substring(headerStartIndex, headerEndIndex);
-
-  return headerValue;
-}
-
-function setHeaderValue(email: string, header: string, value: string) {
-  return email.replace(getHeaderValue(email, header), value);
-}
-
-async function revertForwarderChanges(email: string) {
-  // Google sets their own Message-ID and put the original one in X-Google-Original-Message-ID when forwarding
```

Comment:
I'm worried the new code doesn't have coverage on this due to lack of tests, are you sure your changes work for this DKIM signature and message ID?

Reply:
Unfortunately I dont have a non-sensitive email that contains ARC changes which can be pushed to git. I used the one I have with existing tests and they are passing (after successful modifications)